src/helper/class-ee-site.php

```diff
@@ -2095,7 +2095,8 @@ public function clone( $args, $assoc_args ) {
 			EE::log('Syncing database');
 			copy_site_db( $source, $destination );
 		}
-		EE::success( 'Site cloned successfully' );
+		EE::runcommand(' site info ' . $destination->name);
```

Comment:
Please follow proper spacing conventions as per the rest of the codebase

Reply:
I had to put additional whitespaces because from the error that I got  while executing `site info` (from EE::runcommand) without any spaces seems to indicate that command is  ended  up  like this `site info<sitename>` which is causing error.